### PR TITLE
Static docs.hopsworks.ai landing page

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-www.featureops.ai
\ No newline at end of file
+docs.hopsworks.ai
```
Comment (moritzmeister) [inline CNAME:1]: just leave it like it is, and then github should fix itself when you change the setting.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+## Join the community
+-  Ask questions and give us feedback in the [Hopsworks Community](https://community.hopsworks.ai/)
+- Follow us on [Twitter](https://twitter.com/hopsworks)
+- Check out all our latest [product releases](https://github.com/logicalclocks/hopsworks/releases)
+
+
+## Contribute
+We are building the most complete and modular ML platform available in the market and we count on your support to continuously improve Hopsworks. Feel free to [give us suggestions](https://github.com/logicalclocks/feature-store-api), [report bugs](https://github.com/logicalclocks/feature-store-api/issues) and [add features to our library](https://github.com/logicalclocks/feature-store-api) anytime.
```
Comment (moritzmeister) [inline docs/index.md:49]: are these links correct?

**Review by moritzmeister — CHANGES_REQUESTED** [review]

File: CNAME (hunk 1 above)
Comment (robzor92) [reply CNAME]: reverted

File: docs/index.md (hunk 2 above)
Comment (robzor92) [reply docs/index.md]: I fixed two of them to point to hopsworks instead.

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**